src/app/src/sensorUtils.js

```diff
@@ -17,46 +29,87 @@ export function makeRiverGaugeRequest(id, isApiRequest) {
     return axios.get(url);
 }
 
-function parseCsvString(csvString) {
-    const data = Papa.parse(csvString, { header: true, comments: "#", dynamicTyping: true, skipEmptyLines: true });
-    return data;
+export function parseCsvString(csvString) {
+    const data = Papa.parse(csvString, {
```

Comment:
This could be `const { data } = Papa.parse`. Alternately, this function could also be

```
export const parseCsvString = (csvString) => Papa.parse(csvString, options).data;
```

Reply:
Thanks. Leaving as-is since this is just a Prettier reformatting.